curbs/README.md

```diff
@@ -325,6 +372,96 @@ A Curb Space is represented as a JSON object whose fields are as follows:
 
 [Top][toc]
 
+## Curb Object
+
+Defines individual assets located adjacent to a Curb Space or Curb Zone. Important notes about Curb Objects:
+
+  - Curb Objects can be located anywhere: within, beside, or overlapping with Curb Zones, Spaces, or other Curb Objects
+  - Curb Objects must be related to either a Curb Space or Curb Zone
+  - Curb Objects do not have Curb Policies linked directly to them. Associated Curb Policies can be found by looking at the related Curb Zone (either directly or through the Curb Space)
+  - Unlike Zones and similar to Spaces, Objects may be updated as needed, with a new `curb_object_id` being optionally assigned by the city
+
+A Curb Object is represented as a JSON object whose fields are as follows:
+
+| Name   | Type   | Required/Optional   | Description   |
+| ------ | ------ | ------------------- | ------------- |
+| `curb_object_id` | [UUID][uuid] | Required | The ID of the curb object. |
+| `geometry` | [Point][point] | Required |The spatial location of this curb object location. This can represent the approximate center of the object, or the centroid location of the object, depending on its size and shape. |
```

Comment:
It would be great to have this be more flexible and allow line and polygon for objects like bike racks and bus stops. This would simplify the need for linear distance and max length.

Reply:
This point represents a required field. 

We also have an optional `object_shape` field below which is for polygons. 

I don't think lines work in the context of defining where objects area (though you could argue they do with curb zones, but that's not the scope of this).